api.raml

```diff
@@ -928,3 +977,119 @@ schemas:
           429:
             description: |
               Too many failed attempts for this MFA record recently.
+/method:
+  post:
+    description: Create a new recovery method and send verification email.
+    body:
+      application/json:
+        example: |
+          {
+            "employee_id": "12345",
+            "value": "987654"
```

Comment:
This example of what to `POST` to create a new method seems confusing to me. How would an `employee_id` and a numeric-string `value` be enough for the server to know what to create for the new method?

Reply:
It's the same as the pw-api POST/method except that type is not required since 'email' is the only type.